app/domain/authentication/authn_jwt/create_signing_key_interface.rb

```diff
@@ -60,7 +60,8 @@ def fetch_jwks_uri_signing_key
                                                             logger: Rails.logger,
                                                             fetch_required_secrets: Conjur::FetchRequiredSecrets.new,
                                                             resource_class: ::Resource,
-                                                            http: Net::HTTP)
+                                                            http: Net::HTTP,
```

Comment:
Maybe it should be called http_module/ http_class/ http_lib ?

Reply:
http_lib sounds good